app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= presenter.attribute_to_html(:date_modified, label: t('hyrax.base.show.last_modified')) %>
+<%= presenter.attribute_to_html(:creator, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:contributor, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:subject, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:publisher, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:language, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:identifier, render_as: :linked, search_field: 'identifier_tesim') %>
+<%= presenter.attribute_to_html(:keyword, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:date_created, render_as: :linked, search_field: 'date_created_tesim') %>
+<%= presenter.attribute_to_html(:based_near_label) %>
+<%= presenter.attribute_to_html(:related_url, render_as: :external_link) %>
+<%= presenter.attribute_to_html(:resource_type, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:source) %>
+<%= presenter.attribute_to_html(:rights_statement, render_as: :rights_statement, label: t('hyrax.base.show.rights_statement')) %>
```

Comment:
There is no need to add this. The renderer is already calling i18n with these keys: https://github.com/samvera/hyrax/blob/master/app/renderers/hyrax/renderers/attribute_renderer.rb#L39

Reply:
is it? so I just need to add a `rights_statement` translation?

Reply:
It looks like this may be missing from Hyrax's locale files. I'd expect it to be right here: https://github.com/samvera/hyrax/blob/master/config/locales/hyrax.en.yml#L886

Reply:
Though given what @jcoyne linked, maybe this is unrelated, and we need the key here: https://github.com/samvera/hyrax/blob/master/config/locales/hyrax.en.yml#L22